svc-update/update/simpleUpdate.go

```diff
@@ -70,7 +73,13 @@ func (e *ExternalInterface) SimpleUpdate(taskID string, sessionUserName string,
 	}
 
 	targetList := make(map[string][]string)
-	targetList, err = sortTargetList(updateRequest.Targets)
+	links, err := expandAggregatesCollection(req.SessionToken, updateRequest.Targets)
+	if err != nil {
+		errMsg := "Unable to get links from aggregates: " + err.Error()
+		log.Warn(errMsg)
+		return common.GeneralError(http.StatusInternalServerError, response.InternalError, errMsg, nil, taskInfo)
+	}
+	targetList, err = sortTargetList(links)
```

Comment:
Can we add an empty line to separate the steps?

Reply:
Added